internal/cmd/db_create.go

```diff
@@ -132,211 +35,110 @@ var createCmd = &cobra.Command{
 			return fmt.Errorf("invalid database name: %w", err)
 		}
 
-		config, err := settings.ReadSettings()
-		if err != nil {
-			return err
-		}
-
 		client, err := createTursoClientFromAccessToken(true)
 		if err != nil {
 			return err
 		}
 
-		locationId := locationFlag
-		if locationId == "" {
-			locationId, _ = closestLocation(client)
-		}
-		if !isValidLocation(client, locationId) {
-			return fmt.Errorf("location '%s' is not a valid one", locationId)
-		}
-
-		locations, err := locations(client)
+		group, err := groupFromFlag(client)
 		if err != nil {
 			return err
 		}
 
-		image := "latest"
-		if canaryFlag {
-			image = "canary"
-		}
-
-		extensions := ""
-		if enableExtensionsFlag {
-			extensions = "all"
-		}
-
-		dbFile, err := getDbFile(fromFileFlag)
+		location, err := locationFromFlag(client)
 		if err != nil {
 			return err
 		}
 
-		dbText := ""
-		if fromFileFlag != "" {
-			dbText = fmt.Sprintf(" from file %s", internal.Emph(fromFileFlag))
+		if ok, err := shouldCreateGroup(client, group, location); ok {
+			createGroup(client, group, location)
```

Comment:
⚠️ we are ignoring `createGroup` error 

also, how about we simplify this and avoid `else if`:

```go
	ok, err := shouldCreateGroup(client, group, location)
	if err != nil {
		return err
	}
	if ok {
		if err = createGroup(client, group, location); err != nil {
			return err
		}
	}
```